playbooks/openshift-master/private/config.yml

```diff
@@ -134,6 +129,16 @@
   - name: setup bootstrap settings
     import_tasks: tasks/enable_bootstrap_config.yml
 
+- name: Create sync daemonset
```

Comment:
Just move this task into the play above that's already on oo_first_master ?

Reply:
Created https://github.com/openshift/openshift-ansible/pull/8750